float64 start_x
float64 start_y
float64 goal_x
float64 goal_y
---
nav_msgs/Path path
